pkg/deployments/tasks/20221123-pool-recovery-helper/test/test.fork.ts

```diff
@@ -0,0 +1,101 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+import { ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+import { describeForkTest, impersonate, getForkedNetwork, Task, TaskMode, getSigner } from '../../../src';
+import { setCode } from '@nomicfoundation/hardhat-network-helpers';
+
+describeForkTest('PoolRecoveryHelper', 'mainnet', 15998800, function () {
+  let task: Task;
+
+  let helper: Contract;
+  let operator: SignerWithAddress, admin: SignerWithAddress;
+  let vault: Contract, authorizer: Contract;
+
+  const POOL_STABLE = '0xbD482fFb3E6E50dC1c437557C3Bea2B68f3683Ee'; // From ComposableStablePoolFactory
+  const POOL_WEIGHTED = '0xe340EBfcAA544da8bB1Ee9005F1a346D50Ec422e'; // From WeightedPoolFactory
+
+  before('run task', async () => {
+    task = new Task('20221123-pool-recovery-helper', TaskMode.TEST, getForkedNetwork(hre));
+    await task.run({ force: true });
+    helper = await task.deployedInstance('PoolRecoveryHelper');
+  });
+
+  before('load vault', async () => {
+    const vaultTask = new Task('20210418-vault', TaskMode.READ_ONLY, getForkedNetwork(hre));
+
+    vault = await vaultTask.instanceAt('Vault', await helper.getVault());
+    authorizer = await vaultTask.instanceAt('Authorizer', await vault.getAuthorizer());
```

Comment:
```suggestion
    vault = await vaultTask.deployedInstance('Vault');
    authorizer = await vaultTask.deployedInstance('Authorizer');
```

Reply:
The authorizer is actually a standalone task.